tests/raftstore/test_down_peers.rs

```diff
@@ -77,7 +78,7 @@ fn test_leader_down_and_become_leader_again<T: Simulator>(cluster: &mut Cluster<
 }
 
 fn test_down_peers<T: Simulator>(cluster: &mut Cluster<T>, count: u64) {
-    cluster.cfg.raft_store.max_peer_down_duration = Duration::from_millis(100);
+    cluster.cfg.raft_store.max_peer_down_duration = Duration::from_millis(500);
```

Comment:
Because the election timeout is 250ms, if `max_peer_down_duration` is 100ms, and we isolate the leader, the leader will consider all peers are down.